yarn.lock

```diff
@@ -665,10 +665,24 @@
     "@aws-amplify/platform-core" "^1.0.7"
     zod "^3.22.2"
 
-"@aws-amplify/core@6.3.7":
-  version "6.3.7"
-  resolved "https://registry.yarnpkg.com/@aws-amplify/core/-/core-6.3.7.tgz#b71caaa1726f4a5081170ebd6fa90ce5e3ac604c"
-  integrity sha512-OfUUrhgBW0EyjjXoTORR+vFEtnl8owY1CubId3R2aRPrpg1xi9SDrAT8ZX/o1gUPfGOHkalKBF3xhoQSHNh8Vg==
+"@aws-amplify/core@6.4.0":
+  version "6.4.0"
+  resolved "https://registry.yarnpkg.com/@aws-amplify/core/-/core-6.4.0.tgz#33434d1366681e2073007d6b3d4fcf53fdae3c87"
+  integrity sha512-XdXs9cYlOoH1Gf1Rfamh7fLFcc4EQwinRTDFkNMu7qprk80+enPrZzoDiADkparVia8qY0yhOM6GXVjwrMaJOw==
+  dependencies:
+    "@aws-crypto/sha256-js" "5.2.0"
+    "@aws-sdk/types" "3.398.0"
+    "@smithy/util-hex-encoding" "2.0.0"
+    "@types/uuid" "^9.0.0"
+    js-cookie "^3.0.5"
+    rxjs "^7.8.1"
+    tslib "^2.5.0"
+    uuid "^9.0.0"
+
+"@aws-amplify/core@^6.4.0":
+  version "6.4.1"
```

Comment:
Let's figure out what we need to do make sure there is only one version of `@aws-amplify/core` installed

Reply:
`aws-amplify@6.6.0` uses the exact version `@aws-amplify/core@6.4.0`. 
Since our `aws-amplify` peerDep uses version ^6.6.0 we would need to either specify an exact version for both or an exact version for `@aws-amplify/core`. Not sure which is correct in this scenario

Reply:
Discussed offline and updated `aws-amplify` dep in top-level package.json to use a relative version